### PR TITLE
update docs links for gen2

### PR DESCRIPTION
**Pull request by josefaidt**
*Issue #, if available:*

*Description of changes:*

updates links in readme to point to newer Amplify Gen 2 docs

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
+Note: v2 of Amplify Library for Android (currently GA) is built on top of [the AWS SDK for Kotlin](https://aws.amazon.com/sdk-for-kotlin/). You can access this underlying SDK via [the Escape Hatch from AWS Amplify](https://docs.amplify.aws/android/start/kotlin-coroutines/).
 
-You can also use AWS Amplify with [your existing AWS cloud resources](https://docs.amplify.aws/lib/project-setup/use-existing-resources/q/platform/android/). If you are unable to find features you are looking for in Amplify, please open an issue in [the Amplify Library for Android GitHub repo](https://github.com/aws-amplify/amplify-android/issues/new/choose) and we will be happy to consider you request.
+You can also use AWS Amplify with [your existing AWS cloud resources](https://docs.amplify.aws/react/start/connect-existing-aws-resources/). If you are unable to find features you are looking for in Amplify, please open an issue in [the Amplify Library for Android GitHub repo](https://github.com/aws-amplify/amplify-android/issues/new/choose) and we will be happy to consider you request.
```
Comment (tylerjroach) [inline README.md:13]: Needs to be https://docs.amplify.aws/android/start/connect-existing-aws-resources/ instead of react

File: README.md [hunk 2]
```diff
@@ -6,11 +6,11 @@
...
 
-For new projects, we recommend using the latest v2 version of [AWS Amplify Library for Android](https://docs.amplify.aws/start/q/integration/android/) to quickly implement common app use cases like Authentication, Storage, Push Notifications and more.
+For new projects, we recommend using the latest v2 version of [AWS Amplify Library for Android](https://docs.amplify.aws/android/start/quickstart/) to quickly implement common app use cases like Authentication, Storage, Push Notifications and more.
 
-Note: v2 of Amplify Library for Android (currently GA) is built on top of [the AWS SDK for Kotlin](https://aws.amazon.com/sdk-for-kotlin/). You can access this underlying SDK via [the Escape Hatch from AWS Amplify](https://docs.amplify.aws/lib/project-setup/upgrade-guide/q/platform/android/#escape-hatches).
+Note: v2 of Amplify Library for Android (currently GA) is built on top of [the AWS SDK for Kotlin](https://aws.amazon.com/sdk-for-kotlin/). You can access this underlying SDK via [the Escape Hatch from AWS Amplify](https://docs.amplify.aws/android/start/kotlin-coroutines/).
```
Comment (tylerjroach) [inline README.md:11]: This is not a comparable link. Gen2 is missing reference to using AWS Kotlin SDK escape hatches.

**Review by tylerjroach — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (josefaidt) [reply README.md]: oh nice catch

File: README.md (hunk 2 above)
Comment (josefaidt) [reply README.md]: reverted

**Review by tylerjroach — APPROVED** [review]

**Review by vincetran — APPROVED** [review]

**Pull request merged**